doc/userguide/src/Appendices/CommandLineOptions.rst

```diff
@@ -56,6 +56,8 @@ Command line options for test execution
   --tagstatcombine <tags:title>  Creates `combined statistics based on tags`_.
   --tagdoc <pattern:doc>  Adds `documentation to the specified tags`_.
   --tagstatlink <pattern:link:title>  Adds `external links`_ to the *Statistics by Tag* table.
+  --autoexpandkeywords <name:pattern|tag:pattern>  initially `expand keywords`_
```

Comment:
Not sure is "initially" that good word here. Perhaps "automatically" would be better?

Reply:
agree. done.